### DISCUSSION TITLE
i cant get Build 2208.40000.4.0 in insider fast ?

### DISCUSSION THREAD

**Discussion by m0usaa** [LSPosed/MagiskOnWSALocal#135] [Q&A] [answered]
its just WSA-GApps-pico_2207.40000.8.0 how to get the new version 2208.40000.4.0 in insider fast

**Comment by Howard20181** [comment, marked as answer]
Select Release Type: WIF
![image](https://user-images.githubusercontent.com/40033067/192008357-48345d64-e614-47aa-9323-73e377681670.png)

**Reply by m0usaa** [reply]
already did it with no luck

**Reply by Howard20181** [reply]
Are you using the latest commit?

**Reply by m0usaa** [reply]
yes

**Reply by m0usaa** [reply]
i just deleted the download folder and all works now thanks
but i have another issue 
i cant login to my google account it stucks on welcome screen how to fix ??

![image](https://user-images.githubusercontent.com/69627848/192049369-c65d25c2-1214-49cc-9339-128ad77729af.png)

**Reply by Howard20181** [reply]
The problem of not being able to login has been fixed, please check again if it is the latest commit.

**Reply by m0usaa** [reply]
I do Sudo apt upgrade to to update the commit is it right ?

**Reply by m0usaa** [reply]
On the non root version

**Reply by Howard20181** [reply]
> I do Sudo apt upgrade to to update the commit is it right ?

No. 
`git pull`

**Reply by m0usaa** [reply]
Aha ok 
I just deleted the folder of the commit and reclone again 
Mindedgapps fix this problem ?

**Reply by Howard20181** [reply]
Both OpenGApps and MindThegapps fix the issue

**Reply by m0usaa** [reply]
Thank you very very much

**Reply by m0usaa** [reply]
I typed git pull but it gives me not agit repository
May you tell me how to type it

**Reply by Howard20181** [reply]
You should learn how to use git

**Reply by m0usaa** [reply]
may you tell me ?

**Reply by Howard20181** [reply]
https://docs.github.com/get-started/using-git/getting-changes-from-a-remote-repository

**Reply by m0usaa** [reply]
thanks

**Comment by aardc** [comment]
How to change '*'  to insider fast option? The arrow keys  not responding

**Reply by kamekairos** [reply]
Use tab to select the options and spacebar to tick the checkbox, then tab back and hit ok with enter.

**Reply by aardc** [reply]
Thanks a lot. İts worked.
